rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -0,0 +1,156 @@
+from .participant_summary_dao import ParticipantSummaryDao
+from model.participant_summary import ParticipantSummary
+from participant_enums import EnrollmentStatus
+
+class ParticipantCountsOverTimeService(ParticipantSummaryDao):
+
+  def __init__(self):
+    super(ParticipantSummaryDao, self).__init__(ParticipantSummary)
+
+
+  def get_filtered_results(self, start_date, end_date, filters, stratification='ENROLLMENT_STATUS'):
+    """Queries DB, returns results in format consumed by front-end
+
+    :param start_date: Start date string, e.g. '2018-01-01'
+    :param end_date: End date string, e.g. '2018-01-31'
+    :param filters: Objects representing filters specified in UI
+    :param stratification: How to stratify (layer) results, as in a stacked bar chart
+    :return: Filtered, stratified results by date
+    """
+
+    start_date = start_date.replace('-', '')
+    end_date = end_date.replace('-', '')
+
+    filters_sql = self.get_facets_sql(filters)
+
+    if stratification == 'TOTAL':
+      strata = ['TOTAL']
+      sql = """
```

Comment:
Are filters not supported on the TOTAL stratification?

If not, please add validation logic to that effect in validate_params.

Reply:
Filters are supported for TOTAL, and now implemented.  Thanks for flagging that.